messages_en-2.txt

```diff
@@ -816,33 +983,28 @@ odk_change_language=Change Language
 odk_change_password=Password
 odk_change_server_url=Server URL
 odk_change_username=ODK Username
-odk_clearanswer_confirm=Remove the response to \"\${0}\"?
+odk_clearanswer_confirm=Remove the response to \"%s\"?
 odk_clear_answer=Remove response
 odk_clear_answer_ask=Remove This Response?
 odk_clear_answer_no=Cancel
-odk_completed_data=Complete (\${0})
+odk_completed_data=Complete (%s)
 odk_data=Saved Forms
 odk_data_saved_error=Sorry, form save failed!
 odk_data_saved_ok=Form successfully saved!
 odk_default_completed=Default to Finalized
 odk_default_completed_summary=Mark form as finalized by default
 odk_default_server_url=https://opendatakit.appspot.com
-odk_delete_confirm=Delete \${0} form(s)?
+odk_delete_confirm=Delete %s form(s)?
 odk_delete_file=Delete Selected
 odk_delete_no=Do Not Delete
-odk_delete_repeat=Remove group
-odk_delete_repeat_ask=Remove This Group?
-odk_delete_repeat_confirm=Remove group \"\${0}\" and all of its sub-groups?
-odk_delete_repeat_no=Cancel
 odk_delete_yes=Delete Forms
 odk_discard_answer=Remove response
-odk_discard_group=Remove group
 odk_download=Get Selected
 odk_download_forms_result=Download Result
 odk_downloading_data=Connecting to Server
 odk_download_all_successful=All forms downloaded successfully.
 odk_download_complete=Form Get Complete
-odk_download_failed_with_error=Form \"${0}\" failed. ${1}.
+odk_download_failed_with_error=Form \"%1$s\" failed. %2$s.
```

Comment:
@ctsims  That's not what's in our strings.xml file currently: https://github.com/dimagi/commcare-odk/blob/master/app/res/values/strings.xml#L130-L352